src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -1145,9 +1145,11 @@ namespace CondenserLoopTowers {
 					ShowSevereError( cCurrentModuleObject + " \"" + SimpleTower( TowerNum ).Name + "\". Low-speed nominal capacity must be less than the high-speed nominal capacity." );
 					ErrorsFound = true;
 				}
-				if ( SimpleTower( TowerNum ).TowerFreeConvNomCap >= SimpleTower( TowerNum ).TowerLowSpeedNomCap ) {
-					ShowSevereError( cCurrentModuleObject + " \"" + SimpleTower( TowerNum ).Name + "\". Free convection nominal capacity must be less than the low-speed nominal capacity." );
-					ErrorsFound = true;
+				if ( !SimpleTower( TowerNum ).TowerLowSpeedNomCapWasAutoSized ) {
+					if ( SimpleTower( TowerNum ).TowerFreeConvNomCap >= SimpleTower( TowerNum ).TowerLowSpeedNomCap ) {
+						ShowSevereError( cCurrentModuleObject + " \"" + SimpleTower( TowerNum ).Name + "\". Free convection nominal capacity must be less than the low-speed nominal capacity." );
+						ErrorsFound = true;
+					}
```

Comment:
Yes, I expect the low speed capacity to be higher than the free convection capacity. The default sizing factors (High speed capacity multipliers) used for free convection is 0.1 and 0.5 for low speed.  This defect was fatal because, AutoSize value of -9999.0 of low speed capacity was compared against user specified free convection capacity.